src/Dfc.ProviderPortal.Apprenticeships/Data/StoredProcedures/UpdateRecordStatuses.js

```diff
@@ -57,7 +57,8 @@ function updateSproc(UKPRN, currentStatus, statusToBeChangedTo) {
     function tryUpdate(document) {
         // DocumentDB supports optimistic concurrency control via HTTP ETag.
         var requestOptions = { etag: document._etag };
-        document.RecordStatus = statusToBeChangedTo;
```

Comment:
This line needs to stay, need to add:
`document.ApprenticeshipLocations.forEach(l => l.RecordStatus = statusToBeChangedTo);`

Reply:
changed